onyo/main.py

```diff
@@ -236,6 +236,66 @@ def setup_parser():
     )
     cmd_fsck.set_defaults(run=commands.fsck)
     #
+    # subcommand "get"
+    #
+    cmd_get = subcmds.add_parser(
+        'get',
+        description=textwrap.dedent(commands.get.__doc__),
+        formatter_class=SubcommandHelpFormatter,
+        help='print the requested key(s) for matching assets'
```

Comment:
Maybe `return`? I can change it to `Return matching assets and values corresponding to the requested keys, or the pseudo-key values if no keys are given`. Although I am in favor of omitting the last part, because if no `--keys` are specified, then the default value for that is used (the pseudo-keys). In other words, by not specifying `--keys` the user is requesting all pseudo-keys.